<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reply_feedback_page</class>
 <widget class="QMainWindow" name="reply_feedback_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/backgournds/FlyTranslate-logo/background_image.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtContent">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>601</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Reply title: (no more than 255 characters)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Reply Content:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubmitReply">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Submit Reply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clear Content</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblbtnBack">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>&lt;&lt; Back to Feedback Detail</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Images_for_design/background_imgs.qrc"/>
 </resources>
 <connections/>
</ui>
